### ISSUE TITLE
pcsx2 core is broken for Windows

### ISSUE THREAD

**Issue by Tasosgemah** [hunterk/libretro_builds#4]
It crashes back to the main menu and gives this error in the log:

[ERROR] Failed to open libretro core: "G:\RetroArch\cores\pcsx2_libretro.dll"

[ERROR] Error(s): The specified module could not be found.

**Comment by brad86m** [comment]
Same here. I believe Hunterk is looking into it, though. I did ask on the discord, but of course, you're not allowed to talk about it on there, for the most part.

**Comment by hunterk** [comment]
So, in looking into this, it seems it may just be a lack of vs c++ redist. Do you guys have the latest version installed? I asked someone on discord to try it out and it fired right up for them, and I ran dumpbin to identify all of the core's dependencies, which are:

    WINMM.dll
    ADVAPI32.dll
    D3DCOMPILER_47.dll
    d3d11.dll
    dxgi.dll
    KERNEL32.dll
    api-ms-win-core-memory-l1-1-5.dll
    api-ms-win-core-memory-l1-1-6.dll
    USER32.dll
    VERSION.dll
    MSVCP140D.dll
    IMM32.dll
    VCRUNTIME140D.dll
    VCRUNTIME140_1D.dll
    ucrtbased.dll

The only ones that seem questionable are the MSVCP140D and VCRUNTIME140* libs, which are included in that latest redist.

EDIT: ah, someone on discord gave me the tip: those are the debug libs, which is why you guys don't have them. I should be able to build the 'release' variant to fix that.

**Comment by Tasosgemah** [comment]
Is there any negative in having the debug libs?

**Comment by hunterk** [comment]
No, the logs are just kinda spammy. It shouldn't hurt anything, just takes up some extra space.

I'm not sure why it's always building debug, though. Everything looks to be in order with my build process. It may be just something up with the core as it stands right now.

**Comment by brad86m** [comment]
Happy we have a workaround. Do you suggest installing these debug files, or waiting for you to update ?

**Comment by hunterk** [comment]
if you don't mind installing the debug libs, that's probably the way to go for the short term. I'm not clear on what's causing the debug builds, since I say right in there 'build_type=Release' /shrug

**Comment by Tasosgemah** [comment]
@hunterk 

Not sure what the correct files are to install. Could you provide a link?

**Comment by brad86m** [comment]
I tried the debug install myself, but I still have the same error in RA. I'll wait.

**Comment by brad86m** [comment]
Any news on a build without the debug libs reliance ?
